codecov.yml

```diff
@@ -1,7 +1,4 @@
 coverage:
   status:
     project:
-      default:
-        # basic
-        target: 5%
-        threshold: 10%
+      patch: off
```

Comment:
Move `patch: off` under `status`. Currently you are disabling `codecov/project/patch` (which doesn't exist anyway).